src/one_dragon/utils/cv2_utils.py

```
@@ -1234,3 +1235,50 @@ def is_colorful(img: MatLike, saturation_threshold: int = 30, color_ratio_thresh
+        rgb_diff_threshold: 颜色阈值，RGB只差小于此则认为像素块为灰色
+        percent_rate: 满足条件的点的占比阈值
+
+    Returns:
+        bool: True 表示是彩色的，False 表示接近灰度
+    """
+    if img is None or img.size == 0:
+        return False
+
+    # 裁剪图片
+    if rect is not None:
+        img, crop_rect = crop_image(img, rect)
```

Ruff (0.15.9)

[warning] Unpacked variable `crop_rect` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)